diff -ruN paintdotnet-0.1.63.orig/configure paintdotnet-0.1.63/configure
--- paintdotnet-0.1.63.orig/configure	2009-03-25 09:08:32.000000000 -0700
+++ paintdotnet-0.1.63/configure	2009-05-03 17:55:09.000000000 -0700
@@ -2,6 +2,7 @@
 VERSION=0.1
 PACKAGE=paintdotnet
 prefix=/usr/local
+libdir=/usr/local/lib
 config=RELEASE_AND_PACKAGE_ANY_CPU
 configurations=" DEBUG_ANY_CPU RELEASE_ANY_CPU RELEASE_AND_PACKAGE_ANY_CPU"
 common_packages=""
@@ -9,7 +10,7 @@
 
 usage ()
 {
-	echo "Usage : configure [--prefix=PREFIX] [--config=CONFIG]"
+	echo "Usage : configure [--prefix=PREFIX] [--libdir=LIBDIR] [--config=CONFIG]"
 	echo
 	echo "Configurations available :"
 	for c in $configurations; do
@@ -92,6 +93,13 @@
 				shift
 				prefix=$1
 				;;
+		--libdir=*)
+				libdir=`echo $1 | sed 's/--libdir=//'`
+				;;
+		--libdir)
+				shift
+				libdir=$1
+				;;
 		--config=*)
 				conf=`echo $1 | sed 's/--config=//'`
 				validate_config "$conf"
@@ -122,6 +130,7 @@
 check_svn_revision
 
 echo "prefix=$prefix" > config.make
+echo "libdir=$libdir" >> config.make
 echo "RUNTIME=mono" >> config.make
 echo "ASSEMBLY_VERSION=$VERSION.$revision.0" >> config.make
 echo "VERSION=$VERSION.$revision" >> config.make
@@ -131,5 +140,6 @@
 echo
 echo "$PACKAGE has been configured with "
 echo "        prefix = $prefix"
+echo "        libdir = $libdir"
 echo "        config = $config"
 echo
diff -ruN paintdotnet-0.1.63.orig/Data/data.pc.in paintdotnet-0.1.63/Data/data.pc.in
--- paintdotnet-0.1.63.orig/Data/data.pc.in	2008-09-17 20:13:49.000000000 -0700
+++ paintdotnet-0.1.63/Data/data.pc.in	2009-05-03 17:55:09.000000000 -0700
@@ -3,4 +3,4 @@
 Version: 0.1
 
 Requires: 
-Libs: -r:@prefix@/lib/@PACKAGE@/PaintDotNet.Data.dll
+Libs: -r:@libdir@/@PACKAGE@/PaintDotNet.Data.dll
diff -ruN paintdotnet-0.1.63.orig/Data/Makefile paintdotnet-0.1.63/Data/Makefile
--- paintdotnet-0.1.63.orig/Data/Makefile	2009-03-25 08:25:20.000000000 -0700
+++ paintdotnet-0.1.63/Data/Makefile	2009-05-03 17:55:09.000000000 -0700
@@ -195,44 +195,44 @@
 	resgen2 '$<' '$@'
 
 $(ASSEMBLY) $(ASSEMBLY_MDB): $(build_sources) $(build_resources) $(build_datafiles) $(DLL_REFERENCES) $(PROJECT_REFERENCES) $(build_xamlg_list)
-	make pre-all-local-hook prefix=$(prefix)
+	make pre-all-local-hook prefix=$(prefix) libdir=$(libdir)
 	mkdir -p $(dir $(ASSEMBLY))
 	make $(CONFIG)_BeforeBuild
 	$(ASSEMBLY_COMPILER_COMMAND) $(ASSEMBLY_COMPILER_FLAGS) -out:$(ASSEMBLY) -target:$(COMPILE_TARGET) $(build_sources_embed) $(build_resources_embed) $(build_references_ref)
 	make $(CONFIG)_AfterBuild
-	make post-all-local-hook prefix=$(prefix)
+	make post-all-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(PAINTDOTNET_RESOURCES_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL) $(DATA_PC) $(PDNLIB_DLL_MDB) $(PAINTDOTNET_RESOURCES_DLL) $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(PDNLIB_DLL) $(MICROSOFT_INK_DLL) $(PAINTDOTNET_SYSTEMLAYER_DLL)
-	make pre-install-local-hook prefix=$(prefix)
-	mkdir -p $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || cp $(PAINTDOTNET_RESOURCES_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || cp $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || cp $(PAINTDOTNET_STYLUSREADER_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	mkdir -p $(DESTDIR)$(prefix)/lib/pkgconfig
-	test -z '$(DATA_PC)' || cp $(DATA_PC) $(DESTDIR)$(prefix)/lib/pkgconfig
-	test -z '$(PDNLIB_DLL_MDB)' || cp $(PDNLIB_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || cp $(PAINTDOTNET_RESOURCES_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PDNLIB_DLL)' || cp $(PDNLIB_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(MICROSOFT_INK_DLL)' || cp $(MICROSOFT_INK_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
+	make pre-install-local-hook prefix=$(prefix) libdir=$(libdir)
+	mkdir -p $(DESTDIR)$(libdir)/$(PACKAGE)
+	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || cp $(PAINTDOTNET_RESOURCES_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || cp $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || cp $(PAINTDOTNET_STYLUSREADER_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	mkdir -p $(DESTDIR)$(libdir)/pkgconfig
+	test -z '$(DATA_PC)' || cp $(DATA_PC) $(DESTDIR)$(libdir)/pkgconfig
+	test -z '$(PDNLIB_DLL_MDB)' || cp $(PDNLIB_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || cp $(PAINTDOTNET_RESOURCES_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PDNLIB_DLL)' || cp $(PDNLIB_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(MICROSOFT_INK_DLL)' || cp $(MICROSOFT_INK_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
 	make post-install-local-hook prefix=$(prefix)
 
 uninstall-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(PAINTDOTNET_RESOURCES_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL) $(DATA_PC) $(PDNLIB_DLL_MDB) $(PAINTDOTNET_RESOURCES_DLL) $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(PDNLIB_DLL) $(MICROSOFT_INK_DLL) $(PAINTDOTNET_SYSTEMLAYER_DLL)
 	make pre-uninstall-local-hook prefix=$(prefix)
-	rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY))
-	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
-	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL_MDB))
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL_MDB))
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL))
-	test -z '$(DATA_PC)' || rm -f $(DESTDIR)$(prefix)/lib/pkgconfig/$(notdir $(DATA_PC))
-	test -z '$(PDNLIB_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PDNLIB_DLL_MDB))
-	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL))
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB))
-	test -z '$(PDNLIB_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PDNLIB_DLL))
-	test -z '$(MICROSOFT_INK_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(MICROSOFT_INK_DLL))
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL))
+	rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY))
+	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
+	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL_MDB))
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL_MDB))
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL))
+	test -z '$(DATA_PC)' || rm -f $(DESTDIR)$(libdir)/pkgconfig/$(notdir $(DATA_PC))
+	test -z '$(PDNLIB_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PDNLIB_DLL_MDB))
+	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL))
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB))
+	test -z '$(PDNLIB_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PDNLIB_DLL))
+	test -z '$(MICROSOFT_INK_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(MICROSOFT_INK_DLL))
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL))
 	make post-uninstall-local-hook prefix=$(prefix)
 
 install: install-local
diff -ruN paintdotnet-0.1.63.orig/Effects/effects.pc.in paintdotnet-0.1.63/Effects/effects.pc.in
--- paintdotnet-0.1.63.orig/Effects/effects.pc.in	2008-09-17 20:13:49.000000000 -0700
+++ paintdotnet-0.1.63/Effects/effects.pc.in	2009-05-03 17:55:09.000000000 -0700
@@ -3,4 +3,4 @@
 Version: 0.1
 
 Requires: 
-Libs: -r:@prefix@/lib/@PACKAGE@/PaintDotNet.Effects.dll
+Libs: -r:@libdir@/@PACKAGE@/PaintDotNet.Effects.dll
diff -ruN paintdotnet-0.1.63.orig/Effects/Makefile paintdotnet-0.1.63/Effects/Makefile
--- paintdotnet-0.1.63.orig/Effects/Makefile	2009-03-25 08:26:01.000000000 -0700
+++ paintdotnet-0.1.63/Effects/Makefile	2009-05-03 17:55:09.000000000 -0700
@@ -243,47 +243,47 @@
 	resgen2 '$<' '$@'
 
 $(ASSEMBLY) $(ASSEMBLY_MDB): $(build_sources) $(build_resources) $(build_datafiles) $(DLL_REFERENCES) $(PROJECT_REFERENCES) $(build_xamlg_list)
-	make pre-all-local-hook prefix=$(prefix)
+	make pre-all-local-hook prefix=$(prefix) libdir=$(libdir)
 	mkdir -p $(dir $(ASSEMBLY))
 	make $(CONFIG)_BeforeBuild
 	$(ASSEMBLY_COMPILER_COMMAND) $(ASSEMBLY_COMPILER_FLAGS) -out:$(ASSEMBLY) -target:$(COMPILE_TARGET) $(build_sources_embed) $(build_resources_embed) $(build_references_ref)
 	make $(CONFIG)_AfterBuild
-	make post-all-local-hook prefix=$(prefix)
+	make post-all-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(PAINTDOTNET_SYSTEMLAYER_DLL) $(PAINTDOTNET_DATA_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL) $(EFFECTS_PC) $(PAINTDOTNET_RESOURCES_DLL) $(PDNLIB_DLL_MDB) $(PAINTDOTNET_DATA_DLL) $(PDNLIB_DLL) $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(PAINTDOTNET_RESOURCES_DLL_MDB)
-	make pre-install-local-hook prefix=$(prefix)
-	mkdir -p $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_DATA_DLL_MDB)' || cp $(PAINTDOTNET_DATA_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || cp $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || cp $(PAINTDOTNET_STYLUSREADER_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	mkdir -p $(DESTDIR)$(prefix)/lib/pkgconfig
-	test -z '$(EFFECTS_PC)' || cp $(EFFECTS_PC) $(DESTDIR)$(prefix)/lib/pkgconfig
-	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || cp $(PAINTDOTNET_RESOURCES_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PDNLIB_DLL_MDB)' || cp $(PDNLIB_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_DATA_DLL)' || cp $(PAINTDOTNET_DATA_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PDNLIB_DLL)' || cp $(PDNLIB_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || cp $(PAINTDOTNET_RESOURCES_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	make post-install-local-hook prefix=$(prefix)
+	make pre-install-local-hook prefix=$(prefix) libdir=$(libdir)
+	mkdir -p $(DESTDIR)$(libdir)/$(PACKAGE)
+	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_DATA_DLL_MDB)' || cp $(PAINTDOTNET_DATA_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || cp $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || cp $(PAINTDOTNET_STYLUSREADER_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	mkdir -p $(DESTDIR)$(libdir)/pkgconfig
+	test -z '$(EFFECTS_PC)' || cp $(EFFECTS_PC) $(DESTDIR)$(libdir)/pkgconfig
+	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || cp $(PAINTDOTNET_RESOURCES_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PDNLIB_DLL_MDB)' || cp $(PDNLIB_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_DATA_DLL)' || cp $(PAINTDOTNET_DATA_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PDNLIB_DLL)' || cp $(PDNLIB_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || cp $(PAINTDOTNET_RESOURCES_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	make post-install-local-hook prefix=$(prefix) libdir=$(libdir)
 
 uninstall-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(PAINTDOTNET_SYSTEMLAYER_DLL) $(PAINTDOTNET_DATA_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL) $(EFFECTS_PC) $(PAINTDOTNET_RESOURCES_DLL) $(PDNLIB_DLL_MDB) $(PAINTDOTNET_DATA_DLL) $(PDNLIB_DLL) $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(PAINTDOTNET_RESOURCES_DLL_MDB)
-	make pre-uninstall-local-hook prefix=$(prefix)
-	rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY))
-	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL))
-	test -z '$(PAINTDOTNET_DATA_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_DATA_DLL_MDB))
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL_MDB))
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL))
-	test -z '$(EFFECTS_PC)' || rm -f $(DESTDIR)$(prefix)/lib/pkgconfig/$(notdir $(EFFECTS_PC))
-	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL))
-	test -z '$(PDNLIB_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PDNLIB_DLL_MDB))
-	test -z '$(PAINTDOTNET_DATA_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_DATA_DLL))
-	test -z '$(PDNLIB_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PDNLIB_DLL))
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB))
-	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL_MDB))
-	make post-uninstall-local-hook prefix=$(prefix)
+	make pre-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
+	rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY))
+	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL))
+	test -z '$(PAINTDOTNET_DATA_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_DATA_DLL_MDB))
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL_MDB))
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL))
+	test -z '$(EFFECTS_PC)' || rm -f $(DESTDIR)$(libdir)/pkgconfig/$(notdir $(EFFECTS_PC))
+	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL))
+	test -z '$(PDNLIB_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PDNLIB_DLL_MDB))
+	test -z '$(PAINTDOTNET_DATA_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_DATA_DLL))
+	test -z '$(PDNLIB_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PDNLIB_DLL))
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB))
+	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL_MDB))
+	make post-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install: install-local
 uninstall: uninstall-local
diff -ruN paintdotnet-0.1.63.orig/Makefile.include paintdotnet-0.1.63/Makefile.include
--- paintdotnet-0.1.63.orig/Makefile.include	2008-09-17 20:13:49.000000000 -0700
+++ paintdotnet-0.1.63/Makefile.include	2009-05-03 17:55:09.000000000 -0700
@@ -29,9 +29,9 @@
 pkglib_SCRIPTS = $(ASSEMBLY)
 bin_SCRIPTS = $(BINARIES)
 
-programfilesdir = @prefix@/lib/@PACKAGE@
+programfilesdir = @libdir@/@PACKAGE@
 programfiles_DATA = $(PROGRAMFILES)
-linuxpkgconfigdir = @prefix@/lib/pkgconfig
+linuxpkgconfigdir = @libdir@/pkgconfig
 linuxpkgconfig_DATA = $(LINUX_PKGCONFIG)
 
 
@@ -54,5 +54,5 @@
 	test -z '$3' || chmod +x '$$@'
 
 $2: $2.in $(top_srcdir)/config.make
-	sed -e "s,@prefix@,$(prefix)," -e "s,@PACKAGE@,$(PACKAGE)," < $2.in > $2
-endef
\ No newline at end of file
+	sed -e "s,@prefix@,$(prefix)," -e "s,@PACKAGE@,$(PACKAGE)," -e "s,@libdir@,$(libdir)," < $2.in > $2
+endef
\ No newline at end of file
diff -ruN paintdotnet-0.1.63.orig/paintdotnet.in paintdotnet-0.1.63/paintdotnet.in
--- paintdotnet-0.1.63.orig/paintdotnet.in	2008-09-17 20:13:49.000000000 -0700
+++ paintdotnet-0.1.63/paintdotnet.in	2009-05-03 17:55:09.000000000 -0700
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-exec mono "@prefix@/lib/@PACKAGE@/PaintDotNet.exe" "$@"
+exec mono "@libdir@/@PACKAGE@/PaintDotNet.exe" "$@"
diff -ruN paintdotnet-0.1.63.orig/paintdotnet.make paintdotnet-0.1.63/paintdotnet.make
--- paintdotnet-0.1.63.orig/paintdotnet.make	2009-03-25 07:28:15.000000000 -0700
+++ paintdotnet-0.1.63/paintdotnet.make	2009-05-03 17:56:13.000000000 -0700
@@ -419,48 +419,48 @@
 	resgen2 '$<' '$@'
 
 $(ASSEMBLY) $(ASSEMBLY_MDB): $(build_sources) $(build_resources) $(build_datafiles) $(DLL_REFERENCES) $(PROJECT_REFERENCES) $(build_xamlg_list)
-	make pre-all-local-hook prefix=$(prefix)
+	make pre-all-local-hook prefix=$(prefix) libdir=$(libdir)
 	mkdir -p $(dir $(ASSEMBLY))
 	make $(CONFIG)_BeforeBuild
 	$(ASSEMBLY_COMPILER_COMMAND) $(ASSEMBLY_COMPILER_FLAGS) -out:$(ASSEMBLY) -target:$(COMPILE_TARGET) $(build_sources_embed) $(build_resources_embed) $(build_references_ref)
 	make $(CONFIG)_AfterBuild
-	make post-all-local-hook prefix=$(prefix)
+	make post-all-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(PAINTDOTNET_SYSTEMLAYER_DLL) $(PAINTDOTNET_DATA_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(PAINTDOTNET_EFFECTS_DLL_MDB) $(PAINTDOTNET) $(PAINTDOTNET_STYLUSREADER_DLL) $(PAINTDOTNET_RESOURCES_DLL) $(PDNLIB_DLL_MDB) $(PAINTDOTNET_DATA_DLL) $(PDNLIB_DLL) $(PAINTDOTNET_EFFECTS_DLL) $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(PAINTDOTNET_RESOURCES_DLL_MDB)
-	make pre-install-local-hook prefix=$(prefix)
-	mkdir -p $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_DATA_DLL_MDB)' || cp $(PAINTDOTNET_DATA_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || cp $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_EFFECTS_DLL_MDB)' || cp $(PAINTDOTNET_EFFECTS_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
+	make pre-install-local-hook prefix=$(prefix) libdir=$(libdir)
+	mkdir -p $(DESTDIR)$(libdir)/$(PACKAGE)
+	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_DATA_DLL_MDB)' || cp $(PAINTDOTNET_DATA_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || cp $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_EFFECTS_DLL_MDB)' || cp $(PAINTDOTNET_EFFECTS_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
 	mkdir -p $(DESTDIR)$(prefix)/bin
 	test -z '$(PAINTDOTNET)' || cp $(PAINTDOTNET) $(DESTDIR)$(prefix)/bin
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || cp $(PAINTDOTNET_STYLUSREADER_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || cp $(PAINTDOTNET_RESOURCES_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PDNLIB_DLL_MDB)' || cp $(PDNLIB_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_DATA_DLL)' || cp $(PAINTDOTNET_DATA_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PDNLIB_DLL)' || cp $(PDNLIB_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_EFFECTS_DLL)' || cp $(PAINTDOTNET_EFFECTS_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || cp $(PAINTDOTNET_RESOURCES_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	make post-install-local-hook prefix=$(prefix)
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || cp $(PAINTDOTNET_STYLUSREADER_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || cp $(PAINTDOTNET_RESOURCES_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PDNLIB_DLL_MDB)' || cp $(PDNLIB_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_DATA_DLL)' || cp $(PAINTDOTNET_DATA_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PDNLIB_DLL)' || cp $(PDNLIB_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_EFFECTS_DLL)' || cp $(PAINTDOTNET_EFFECTS_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || cp $(PAINTDOTNET_RESOURCES_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	make post-install-local-hook prefix=$(prefix) libdir=$(libdir)
 
 uninstall-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(PAINTDOTNET_SYSTEMLAYER_DLL) $(PAINTDOTNET_DATA_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(PAINTDOTNET_EFFECTS_DLL_MDB) $(PAINTDOTNET) $(PAINTDOTNET_STYLUSREADER_DLL) $(PAINTDOTNET_RESOURCES_DLL) $(PDNLIB_DLL_MDB) $(PAINTDOTNET_DATA_DLL) $(PDNLIB_DLL) $(PAINTDOTNET_EFFECTS_DLL) $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(PAINTDOTNET_RESOURCES_DLL_MDB)
-	make pre-uninstall-local-hook prefix=$(prefix)
-	rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY))
-	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL))
-	test -z '$(PAINTDOTNET_DATA_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_DATA_DLL_MDB))
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL_MDB))
-	test -z '$(PAINTDOTNET_EFFECTS_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_EFFECTS_DLL_MDB))
+	make pre-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
+	rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY))
+	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL))
+	test -z '$(PAINTDOTNET_DATA_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_DATA_DLL_MDB))
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL_MDB))
+	test -z '$(PAINTDOTNET_EFFECTS_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_EFFECTS_DLL_MDB))
 	test -z '$(PAINTDOTNET)' || rm -f $(DESTDIR)$(prefix)/bin/$(notdir $(PAINTDOTNET))
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL))
-	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL))
-	test -z '$(PDNLIB_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PDNLIB_DLL_MDB))
-	test -z '$(PAINTDOTNET_DATA_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_DATA_DLL))
-	test -z '$(PDNLIB_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PDNLIB_DLL))
-	test -z '$(PAINTDOTNET_EFFECTS_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_EFFECTS_DLL))
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB))
-	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL_MDB))
-	make post-uninstall-local-hook prefix=$(prefix)
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL))
+	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL))
+	test -z '$(PDNLIB_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PDNLIB_DLL_MDB))
+	test -z '$(PAINTDOTNET_DATA_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_DATA_DLL))
+	test -z '$(PDNLIB_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PDNLIB_DLL))
+	test -z '$(PAINTDOTNET_EFFECTS_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_EFFECTS_DLL))
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB))
+	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL_MDB))
+	make post-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
diff -ruN paintdotnet-0.1.63.orig/PdnLib/Makefile paintdotnet-0.1.63/PdnLib/Makefile
--- paintdotnet-0.1.63.orig/PdnLib/Makefile	2009-03-25 08:26:42.000000000 -0700
+++ paintdotnet-0.1.63/PdnLib/Makefile	2009-05-03 17:55:09.000000000 -0700
@@ -268,39 +268,39 @@
 	resgen2 '$<' '$@'
 
 $(ASSEMBLY) $(ASSEMBLY_MDB): $(build_sources) $(build_resources) $(build_datafiles) $(DLL_REFERENCES) $(PROJECT_REFERENCES) $(build_xamlg_list)
-	make pre-all-local-hook prefix=$(prefix)
+	make pre-all-local-hook prefix=$(prefix) libdir=$(libdir)
 	mkdir -p $(dir $(ASSEMBLY))
 	make $(CONFIG)_BeforeBuild
 	$(ASSEMBLY_COMPILER_COMMAND) $(ASSEMBLY_COMPILER_FLAGS) -out:$(ASSEMBLY) -target:$(COMPILE_TARGET) $(build_sources_embed) $(build_resources_embed) $(build_references_ref)
 	make $(CONFIG)_AfterBuild
-	make post-all-local-hook prefix=$(prefix)
+	make post-all-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(PAINTDOTNET_RESOURCES_DLL_MDB) $(PDNLIB_PC) $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL) $(PAINTDOTNET_RESOURCES_DLL) $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(PAINTDOTNET_SYSTEMLAYER_DLL)
-	make pre-install-local-hook prefix=$(prefix)
-	mkdir -p $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || cp $(PAINTDOTNET_RESOURCES_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	mkdir -p $(DESTDIR)$(prefix)/lib/pkgconfig
-	test -z '$(PDNLIB_PC)' || cp $(PDNLIB_PC) $(DESTDIR)$(prefix)/lib/pkgconfig
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || cp $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || cp $(PAINTDOTNET_STYLUSREADER_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || cp $(PAINTDOTNET_RESOURCES_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	make post-install-local-hook prefix=$(prefix)
+	make pre-install-local-hook prefix=$(prefix) libdir=$(libdir)
+	mkdir -p $(DESTDIR)$(libdir)/$(PACKAGE)
+	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || cp $(PAINTDOTNET_RESOURCES_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	mkdir -p $(DESTDIR)$(libdir)/pkgconfig
+	test -z '$(PDNLIB_PC)' || cp $(PDNLIB_PC) $(DESTDIR)$(libdir)/pkgconfig
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || cp $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || cp $(PAINTDOTNET_STYLUSREADER_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || cp $(PAINTDOTNET_RESOURCES_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	make post-install-local-hook prefix=$(prefix) libdir=$(libdir)
 
 uninstall-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(PAINTDOTNET_RESOURCES_DLL_MDB) $(PDNLIB_PC) $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL) $(PAINTDOTNET_RESOURCES_DLL) $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(PAINTDOTNET_SYSTEMLAYER_DLL)
-	make pre-uninstall-local-hook prefix=$(prefix)
-	rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY))
-	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
-	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL_MDB))
-	test -z '$(PDNLIB_PC)' || rm -f $(DESTDIR)$(prefix)/lib/pkgconfig/$(notdir $(PDNLIB_PC))
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL_MDB))
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL))
-	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL))
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB))
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL))
-	make post-uninstall-local-hook prefix=$(prefix)
+	make pre-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
+	rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY))
+	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
+	test -z '$(PAINTDOTNET_RESOURCES_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL_MDB))
+	test -z '$(PDNLIB_PC)' || rm -f $(DESTDIR)$(libdir)/pkgconfig/$(notdir $(PDNLIB_PC))
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL_MDB))
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL))
+	test -z '$(PAINTDOTNET_RESOURCES_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_RESOURCES_DLL))
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB))
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL))
+	make post-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install: install-local
 uninstall: uninstall-local
diff -ruN paintdotnet-0.1.63.orig/PdnLib/pdnlib.pc.in paintdotnet-0.1.63/PdnLib/pdnlib.pc.in
--- paintdotnet-0.1.63.orig/PdnLib/pdnlib.pc.in	2008-09-17 20:13:49.000000000 -0700
+++ paintdotnet-0.1.63/PdnLib/pdnlib.pc.in	2009-05-03 17:55:09.000000000 -0700
@@ -3,4 +3,4 @@
 Version: 0.1
 
 Requires: 
-Libs: -r:@prefix@/lib/@PACKAGE@/PdnLib.dll
+Libs: -r:@libdir@/@PACKAGE@/PdnLib.dll
diff -ruN paintdotnet-0.1.63.orig/PdnRepair/Makefile paintdotnet-0.1.63/PdnRepair/Makefile
--- paintdotnet-0.1.63.orig/PdnRepair/Makefile	2009-03-25 08:04:18.000000000 -0700
+++ paintdotnet-0.1.63/PdnRepair/Makefile	2009-05-03 17:55:09.000000000 -0700
@@ -90,27 +90,27 @@
 	resgen2 '$<' '$@'
 
 $(ASSEMBLY) $(ASSEMBLY_MDB): $(build_sources) $(build_resources) $(build_datafiles) $(DLL_REFERENCES) $(PROJECT_REFERENCES) $(build_xamlg_list)
-	make pre-all-local-hook prefix=$(prefix)
+	make pre-all-local-hook prefix=$(prefix) libdir=$(libdir)
 	mkdir -p $(dir $(ASSEMBLY))
 	make $(CONFIG)_BeforeBuild
 	$(ASSEMBLY_COMPILER_COMMAND) $(ASSEMBLY_COMPILER_FLAGS) -out:$(ASSEMBLY) -target:$(COMPILE_TARGET) $(build_sources_embed) $(build_resources_embed) $(build_references_ref)
 	make $(CONFIG)_AfterBuild
-	make post-all-local-hook prefix=$(prefix)
+	make post-all-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(PDNREPAIR)
-	make pre-install-local-hook prefix=$(prefix)
-	mkdir -p $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
+	make pre-install-local-hook prefix=$(prefix) libdir=$(libdir)
+	mkdir -p $(DESTDIR)$(libdir)/$(PACKAGE)
+	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
 	mkdir -p $(DESTDIR)$(prefix)/bin
 	test -z '$(PDNREPAIR)' || cp $(PDNREPAIR) $(DESTDIR)$(prefix)/bin
-	make post-install-local-hook prefix=$(prefix)
+	make post-install-local-hook prefix=$(prefix) libdir=$(libdir)
 
 uninstall-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(PDNREPAIR)
-	make pre-uninstall-local-hook prefix=$(prefix)
-	rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY))
-	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
+	make pre-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
+	rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY))
+	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
 	test -z '$(PDNREPAIR)' || rm -f $(DESTDIR)$(prefix)/bin/$(notdir $(PDNREPAIR))
-	make post-uninstall-local-hook prefix=$(prefix)
+	make post-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install: install-local
 uninstall: uninstall-local
diff -ruN paintdotnet-0.1.63.orig/PdnRepair/pdnrepair.in paintdotnet-0.1.63/PdnRepair/pdnrepair.in
--- paintdotnet-0.1.63.orig/PdnRepair/pdnrepair.in	2008-09-17 20:13:49.000000000 -0700
+++ paintdotnet-0.1.63/PdnRepair/pdnrepair.in	2009-05-03 17:55:09.000000000 -0700
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-exec mono "@prefix@/lib/@PACKAGE@/PdnRepair.exe" "$@"
+exec mono "@libdir@/@PACKAGE@/PdnRepair.exe" "$@"
diff -ruN paintdotnet-0.1.63.orig/Resources/Makefile paintdotnet-0.1.63/Resources/Makefile
--- paintdotnet-0.1.63.orig/Resources/Makefile	2009-03-25 08:27:14.000000000 -0700
+++ paintdotnet-0.1.63/Resources/Makefile	2009-05-03 17:55:09.000000000 -0700
@@ -333,35 +333,35 @@
 	resgen2 '$<' '$@'
 
 $(ASSEMBLY) $(ASSEMBLY_MDB): $(build_sources) $(build_resources) $(build_datafiles) $(DLL_REFERENCES) $(PROJECT_REFERENCES) $(build_xamlg_list)
-	make pre-all-local-hook prefix=$(prefix)
+	make pre-all-local-hook prefix=$(prefix) libdir=$(libdir)
 	mkdir -p $(dir $(ASSEMBLY))
 	make $(CONFIG)_BeforeBuild
 	$(ASSEMBLY_COMPILER_COMMAND) $(ASSEMBLY_COMPILER_FLAGS) -out:$(ASSEMBLY) -target:$(COMPILE_TARGET) $(build_sources_embed) $(build_resources_embed) $(build_references_ref)
 	make $(CONFIG)_AfterBuild
-	make post-all-local-hook prefix=$(prefix)
+	make post-all-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL) $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(RESOURCES_PC) $(PAINTDOTNET_SYSTEMLAYER_DLL)
-	make pre-install-local-hook prefix=$(prefix)
-	mkdir -p $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || cp $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || cp $(PAINTDOTNET_STYLUSREADER_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	mkdir -p $(DESTDIR)$(prefix)/lib/pkgconfig
-	test -z '$(RESOURCES_PC)' || cp $(RESOURCES_PC) $(DESTDIR)$(prefix)/lib/pkgconfig
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	make post-install-local-hook prefix=$(prefix)
+	make pre-install-local-hook prefix=$(prefix) libdir=$(libdir)
+	mkdir -p $(DESTDIR)$(libdir)/$(PACKAGE)
+	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || cp $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || cp $(PAINTDOTNET_STYLUSREADER_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	mkdir -p $(DESTDIR)$(libdir)/pkgconfig
+	test -z '$(RESOURCES_PC)' || cp $(RESOURCES_PC) $(DESTDIR)$(libdir)/pkgconfig
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || cp $(PAINTDOTNET_SYSTEMLAYER_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	make post-install-local-hook prefix=$(prefix) libdir=$(libdir)
 
 uninstall-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL) $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB) $(RESOURCES_PC) $(PAINTDOTNET_SYSTEMLAYER_DLL)
-	make pre-uninstall-local-hook prefix=$(prefix)
-	rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY))
-	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL_MDB))
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL))
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB))
-	test -z '$(RESOURCES_PC)' || rm -f $(DESTDIR)$(prefix)/lib/pkgconfig/$(notdir $(RESOURCES_PC))
-	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL))
-	make post-uninstall-local-hook prefix=$(prefix)
+	make pre-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
+	rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY))
+	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL_MDB))
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL))
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL_MDB))
+	test -z '$(RESOURCES_PC)' || rm -f $(DESTDIR)$(libdir)/pkgconfig/$(notdir $(RESOURCES_PC))
+	test -z '$(PAINTDOTNET_SYSTEMLAYER_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_SYSTEMLAYER_DLL))
+	make post-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install: install-local
 uninstall: uninstall-local
diff -ruN paintdotnet-0.1.63.orig/Resources/resources.pc.in paintdotnet-0.1.63/Resources/resources.pc.in
--- paintdotnet-0.1.63.orig/Resources/resources.pc.in	2008-09-17 20:13:49.000000000 -0700
+++ paintdotnet-0.1.63/Resources/resources.pc.in	2009-05-03 17:55:09.000000000 -0700
@@ -3,4 +3,4 @@
 Version: 0.1
 
 Requires: 
-Libs: -r:@prefix@/lib/@PACKAGE@/PaintDotNet.Resources.dll
+Libs: -r:@libdir@/@PACKAGE@/PaintDotNet.Resources.dll
diff -ruN paintdotnet-0.1.63.orig/Resources.mui/custom-hooks.make paintdotnet-0.1.63/Resources.mui/custom-hooks.make
--- paintdotnet-0.1.63.orig/Resources.mui/custom-hooks.make	2008-09-17 20:13:49.000000000 -0700
+++ paintdotnet-0.1.63/Resources.mui/custom-hooks.make	2009-05-03 17:55:09.000000000 -0700
@@ -1,4 +1,4 @@
 pre-install-local-hook:
-	mkdir -p $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	cp PaintDotNet.Strings*.resources $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	cp ../bin/*/PaintDotNet.Strings.3.resources $(DESTDIR)$(prefix)/lib/$(PACKAGE)
+	mkdir -p $(DESTDIR)$(libdir)/$(PACKAGE)
+	cp PaintDotNet.Strings*.resources $(DESTDIR)$(libdir)/$(PACKAGE)
+	cp ../bin/*/PaintDotNet.Strings.3.resources $(DESTDIR)$(libdir)/$(PACKAGE)
diff -ruN paintdotnet-0.1.63.orig/Resources.mui/Makefile paintdotnet-0.1.63/Resources.mui/Makefile
--- paintdotnet-0.1.63.orig/Resources.mui/Makefile	2009-03-25 08:06:25.000000000 -0700
+++ paintdotnet-0.1.63/Resources.mui/Makefile	2009-05-03 17:55:09.000000000 -0700
@@ -110,27 +110,27 @@
 	resgen2 '$<' '$@'
 
 $(ASSEMBLY) $(ASSEMBLY_MDB): $(build_sources) $(build_resources) $(build_datafiles) $(DLL_REFERENCES) $(PROJECT_REFERENCES) $(build_xamlg_list)
-	make pre-all-local-hook prefix=$(prefix)
+	make pre-all-local-hook prefix=$(prefix) libdir=$(libdir)
 	mkdir -p $(dir $(ASSEMBLY))
 	make $(CONFIG)_BeforeBuild
 	$(ASSEMBLY_COMPILER_COMMAND) $(ASSEMBLY_COMPILER_FLAGS) -out:$(ASSEMBLY) -target:$(COMPILE_TARGET) $(build_sources_embed) $(build_resources_embed) $(build_references_ref)
 	make $(CONFIG)_AfterBuild
-	make post-all-local-hook prefix=$(prefix)
+	make post-all-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(RESOURCES_MUI)
-	make pre-install-local-hook prefix=$(prefix)
-	mkdir -p $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
+	make pre-install-local-hook prefix=$(prefix) libdir=$(libdir)
+	mkdir -p $(DESTDIR)$(libdir)/$(PACKAGE)
+	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
 	mkdir -p $(DESTDIR)$(prefix)/bin
 	test -z '$(RESOURCES_MUI)' || cp $(RESOURCES_MUI) $(DESTDIR)$(prefix)/bin
-	make post-install-local-hook prefix=$(prefix)
+	make post-install-local-hook prefix=$(prefix) libdir=$(libdir)
 
 uninstall-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(RESOURCES_MUI)
-	make pre-uninstall-local-hook prefix=$(prefix)
-	rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY))
-	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
+	make pre-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
+	rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY))
+	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
 	test -z '$(RESOURCES_MUI)' || rm -f $(DESTDIR)$(prefix)/bin/$(notdir $(RESOURCES_MUI))
-	make post-uninstall-local-hook prefix=$(prefix)
+	make post-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install: install-local
 uninstall: uninstall-local
diff -ruN paintdotnet-0.1.63.orig/Resources.mui/resources.mui.in paintdotnet-0.1.63/Resources.mui/resources.mui.in
--- paintdotnet-0.1.63.orig/Resources.mui/resources.mui.in	2008-09-17 20:13:49.000000000 -0700
+++ paintdotnet-0.1.63/Resources.mui/resources.mui.in	2009-05-03 17:55:09.000000000 -0700
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-exec mono "@prefix@/lib/@PACKAGE@/Resources.mui.exe" "$@"
+exec mono "@libdir@/@PACKAGE@/Resources.mui.exe" "$@"
diff -ruN paintdotnet-0.1.63.orig/StylusReader/Makefile paintdotnet-0.1.63/StylusReader/Makefile
--- paintdotnet-0.1.63.orig/StylusReader/Makefile	2009-03-25 08:14:44.000000000 -0700
+++ paintdotnet-0.1.63/StylusReader/Makefile	2009-05-03 17:55:09.000000000 -0700
@@ -94,27 +94,27 @@
 	resgen2 '$<' '$@'
 
 $(ASSEMBLY) $(ASSEMBLY_MDB): $(build_sources) $(build_resources) $(build_datafiles) $(DLL_REFERENCES) $(PROJECT_REFERENCES) $(build_xamlg_list)
-	make pre-all-local-hook prefix=$(prefix)
+	make pre-all-local-hook prefix=$(prefix) libdir=$(libdir)
 	mkdir -p $(dir $(ASSEMBLY))
 	make $(CONFIG)_BeforeBuild
 	$(ASSEMBLY_COMPILER_COMMAND) $(ASSEMBLY_COMPILER_FLAGS) -out:$(ASSEMBLY) -target:$(COMPILE_TARGET) $(build_sources_embed) $(build_resources_embed) $(build_references_ref)
 	make $(CONFIG)_AfterBuild
-	make post-all-local-hook prefix=$(prefix)
+	make post-all-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(STYLUSREADER_PC)
-	make pre-install-local-hook prefix=$(prefix)
-	mkdir -p $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	mkdir -p $(DESTDIR)$(prefix)/lib/pkgconfig
-	test -z '$(STYLUSREADER_PC)' || cp $(STYLUSREADER_PC) $(DESTDIR)$(prefix)/lib/pkgconfig
-	make post-install-local-hook prefix=$(prefix)
+	make pre-install-local-hook prefix=$(prefix) libdir=$(libdir)
+	mkdir -p $(DESTDIR)$(libdir)/$(PACKAGE)
+	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	mkdir -p $(DESTDIR)$(libdir)/pkgconfig
+	test -z '$(STYLUSREADER_PC)' || cp $(STYLUSREADER_PC) $(DESTDIR)$(libdir)/pkgconfig
+	make post-install-local-hook prefix=$(prefix) libdir=$(libdir)
 
 uninstall-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(STYLUSREADER_PC)
-	make pre-uninstall-local-hook prefix=$(prefix)
-	rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY))
-	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
-	test -z '$(STYLUSREADER_PC)' || rm -f $(DESTDIR)$(prefix)/lib/pkgconfig/$(notdir $(STYLUSREADER_PC))
-	make post-uninstall-local-hook prefix=$(prefix)
+	make pre-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
+	rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY))
+	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
+	test -z '$(STYLUSREADER_PC)' || rm -f $(DESTDIR)$(libdir)/pkgconfig/$(notdir $(STYLUSREADER_PC))
+	make post-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install: install-local
 uninstall: uninstall-local
diff -ruN paintdotnet-0.1.63.orig/StylusReader/stylusreader.pc.in paintdotnet-0.1.63/StylusReader/stylusreader.pc.in
--- paintdotnet-0.1.63.orig/StylusReader/stylusreader.pc.in	2008-09-17 20:13:49.000000000 -0700
+++ paintdotnet-0.1.63/StylusReader/stylusreader.pc.in	2009-05-03 17:55:09.000000000 -0700
@@ -3,4 +3,4 @@
 Version: 0.1
 
 Requires: 
-Libs: -r:@prefix@/lib/@PACKAGE@/PaintDotNet.StylusReader.dll
+Libs: -r:@libdir@/@PACKAGE@/PaintDotNet.StylusReader.dll
diff -ruN paintdotnet-0.1.63.orig/SystemLayer/Makefile paintdotnet-0.1.63/SystemLayer/Makefile
--- paintdotnet-0.1.63.orig/SystemLayer/Makefile	2009-03-25 08:24:14.000000000 -0700
+++ paintdotnet-0.1.63/SystemLayer/Makefile	2009-05-03 17:55:09.000000000 -0700
@@ -153,33 +153,33 @@
 	resgen2 '$<' '$@'
 
 $(ASSEMBLY) $(ASSEMBLY_MDB): $(build_sources) $(build_resources) $(build_datafiles) $(DLL_REFERENCES) $(PROJECT_REFERENCES) $(build_xamlg_list)
-	make pre-all-local-hook prefix=$(prefix)
+	make pre-all-local-hook prefix=$(prefix) libdir=$(libdir)
 	mkdir -p $(dir $(ASSEMBLY))
 	make $(CONFIG)_BeforeBuild
 	$(ASSEMBLY_COMPILER_COMMAND) $(ASSEMBLY_COMPILER_FLAGS) -out:$(ASSEMBLY) -target:$(COMPILE_TARGET) $(build_sources_embed) $(build_resources_embed) $(build_references_ref)
 	make $(CONFIG)_AfterBuild
-	make post-all-local-hook prefix=$(prefix)
+	make post-all-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL) $(SYSTEMLAYER_PC) $(MICROSOFT_INK_DLL)
-	make pre-install-local-hook prefix=$(prefix)
-	mkdir -p $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || cp $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || cp $(PAINTDOTNET_STYLUSREADER_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	mkdir -p $(DESTDIR)$(prefix)/lib/pkgconfig
-	test -z '$(SYSTEMLAYER_PC)' || cp $(SYSTEMLAYER_PC) $(DESTDIR)$(prefix)/lib/pkgconfig
-	test -z '$(MICROSOFT_INK_DLL)' || cp $(MICROSOFT_INK_DLL) $(DESTDIR)$(prefix)/lib/$(PACKAGE)
-	make post-install-local-hook prefix=$(prefix)
+	make pre-install-local-hook prefix=$(prefix) libdir=$(libdir)
+	mkdir -p $(DESTDIR)$(libdir)/$(PACKAGE)
+	cp $(ASSEMBLY) $(ASSEMBLY_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || cp $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(DESTDIR)$(libdir)/$(PACKAGE)
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || cp $(PAINTDOTNET_STYLUSREADER_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	mkdir -p $(DESTDIR)$(libdir)/pkgconfig
+	test -z '$(SYSTEMLAYER_PC)' || cp $(SYSTEMLAYER_PC) $(DESTDIR)$(libdir)/pkgconfig
+	test -z '$(MICROSOFT_INK_DLL)' || cp $(MICROSOFT_INK_DLL) $(DESTDIR)$(libdir)/$(PACKAGE)
+	make post-install-local-hook prefix=$(prefix) libdir=$(libdir)
 
 uninstall-local: $(ASSEMBLY) $(ASSEMBLY_MDB) $(PAINTDOTNET_STYLUSREADER_DLL_MDB) $(PAINTDOTNET_STYLUSREADER_DLL) $(SYSTEMLAYER_PC) $(MICROSOFT_INK_DLL)
-	make pre-uninstall-local-hook prefix=$(prefix)
-	rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY))
-	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL_MDB))
-	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL))
-	test -z '$(SYSTEMLAYER_PC)' || rm -f $(DESTDIR)$(prefix)/lib/pkgconfig/$(notdir $(SYSTEMLAYER_PC))
-	test -z '$(MICROSOFT_INK_DLL)' || rm -f $(DESTDIR)$(prefix)/lib/$(PACKAGE)/$(notdir $(MICROSOFT_INK_DLL))
-	make post-uninstall-local-hook prefix=$(prefix)
+	make pre-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
+	rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY))
+	test -z '$(ASSEMBLY_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(ASSEMBLY_MDB))
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL_MDB)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL_MDB))
+	test -z '$(PAINTDOTNET_STYLUSREADER_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(PAINTDOTNET_STYLUSREADER_DLL))
+	test -z '$(SYSTEMLAYER_PC)' || rm -f $(DESTDIR)$(libdir)/pkgconfig/$(notdir $(SYSTEMLAYER_PC))
+	test -z '$(MICROSOFT_INK_DLL)' || rm -f $(DESTDIR)$(libdir)/$(PACKAGE)/$(notdir $(MICROSOFT_INK_DLL))
+	make post-uninstall-local-hook prefix=$(prefix) libdir=$(libdir)
 
 install: install-local
 uninstall: uninstall-local
diff -ruN paintdotnet-0.1.63.orig/SystemLayer/systemlayer.pc.in paintdotnet-0.1.63/SystemLayer/systemlayer.pc.in
--- paintdotnet-0.1.63.orig/SystemLayer/systemlayer.pc.in	2008-09-17 20:13:49.000000000 -0700
+++ paintdotnet-0.1.63/SystemLayer/systemlayer.pc.in	2009-05-03 17:55:09.000000000 -0700
@@ -3,4 +3,4 @@
 Version: 0.1
 
 Requires: 
-Libs: -r:@prefix@/lib/@PACKAGE@/PaintDotNet.SystemLayer.dll
+Libs: -r:@libdir@/@PACKAGE@/PaintDotNet.SystemLayer.dll

